### DISCUSSION TITLE
"Model" in measurement table

### DISCUSSION THREAD

**Discussion by Canela352** [dbarzin/deming#215] [General]
Hello Mr Barzin, Deming is a greta tool.
Tn the table "edit measurement" there is a space "model" my question is the meaning in this case or it pursose? or which type of info is collected in this space?

thank you and best reagards

**Comment by Canela352** [comment]
Same question for "Scope"

Thank you adn best regards

**Comment by dbarzin** [comment]
Thank you for your question and for your kind words about Deming.

The "Model" field is used to define the criteria for evaluating the status of a measurement. This field should include the formula or thresholds that indicate whether the measurement is considered "green," "orange," or "red." For example, you might set it as: "green" = 0, "orange" ≥ 1, "red" ≥ 2. This helps in quickly assessing the results based on predefined conditions.

The "Scope" field is intended for cases where the same measurement needs to be applied across different subjects or areas. For instance, you might want to evaluate the implementation of physical security measures at various locations or monitor the signing of non-disclosure agreements (NDAs) by different service providers. This field ensures that you can specify and track measurements relevant to distinct scopes.

I hope this clarifies the purpose of these fields. Please let me know if you need further information!
